src/lwc/healthCheckRun/healthCheckRun.html

```diff
@@ -0,0 +1,36 @@
+<template>
+    <lightning-card title={labelReference.stgEDASettingsHealthCheckTitle}>
+        <div class="slds-grid slds-wrap">
+            <div class="slds-col 
+                        slds-size--2-of-2 
+                        slds-p-horizontal_small 
+                        slds-p-vertical_small
+                        slds-text-body_regular">
+                <p>{labelReference.stgHealthCheckDescription}</p>
+            </div>
+
+            <div class="slds-col 
+                        slds-medium-size--1-of-2 
```

Comment:
Please replace this with slds-size--1-of-2 in both locations so it works on all screen sizes.